srcpkgs/nerd-fonts/template

```diff
@@ -17,19 +17,31 @@ do_install() {
 	vmkdir usr/lib/NerdFonts
 
 	# get all patched fonts
-	find patched-fonts -name '*.otf' -exec install -Dm644 '{}' ${DESTDIR}/usr/share/fonts/NerdFonts/otf \;
-	find patched-fonts -name '*.ttf' -exec install -Dm644 '{}' ${DESTDIR}/usr/share/fonts/NerdFonts/ttf \;
+	find patched-fonts -name '*.otf' -exec install -Dm644 '{}' "${DESTDIR}/usr/share/fonts/NerdFonts/otf" \;
+	find patched-fonts -name '*.ttf' -exec install -Dm644 '{}' "${DESTDIR}/usr/share/fonts/NerdFonts/ttf" \;
+	rm "${DESTDIR}"/usr/share/fonts/NerdFonts/ttf/SymbolsNerdFont*.ttf
 
 	# install patching scripts
 	for sh in bin/scripts/lib/i_*.sh; do
-		vinstall $sh 744 usr/lib/NerdFonts
+		vinstall "$sh" 744 usr/lib/NerdFonts
```

Comment:
these are NOT patching scripts.  this is the list of all glyphs for the cheat-sheet. see [here](https://github.com/ryanoasis/nerd-fonts/tree/master/bin/scripts/lib). they're useless. please remove these.

Reply:
I'll leave that up to the maintainer.

also please chill out

Reply:
I was planning on replacing this with the real patching script on the next release. It can be fixed now, if you'd prefer.

Reply:
if you have plans I'll leave it as is for you